pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -296,6 +272,22 @@ public LocalBrokerData deserialize(String key, byte[] content) throws Exception
         loadSheddingPipeline.add(createLoadSheddingStrategy());
     }
 
+    @Override
+    public void accept(Notification t) {
+        if (t.getPath().startsWith(LoadManager.LOADBALANCE_BROKERS_ROOT)) {
```

Comment:
thinking out loud:
we should have some simpler way to filter out the notifications
now when you register a Listener it is receiving all of the notifications

a better API would be:
` pulsar.getLocalMetadataStore().registerListener(this, (Notification t) -> t.getPath().startsWith(LoadManager.LOADBALANCE_BROKERS_ROOT));`
 
 with an utility for "startsWith":
` pulsar.getLocalMetadataStore().registerListener(this,LoadManager.LOADBALANCE_BROKERS_ROOT));`

Reply:
Yes, it could be improved, we should have 2 basic ways, either exact or prefix match